ethers-solc/src/compile.rs

```diff
@@ -87,6 +112,74 @@ impl Solc {
         Ok(solc)
     }
 
+    /// Assuming the `versions` array is sorted, it returns the first element which satisfies
```

Comment:
Assuming versions are sorted asc, right?
```suggestion
    /// Assuming the `versions` array is sorted, it returns the latest element which satisfies
```